extras/mainnet.json

```
@@ -205,5 +206,11 @@
     },
     "safe_harbor": {
         "registry": "0x8f72fcf695523A6FC7DD97EafDd7A083c386b7b6"
+    },
+    "stake_dao": {
+        "delegation_address": "0x52ea58f4FC3CEd48fa18E909226c1f8A0EF887DC",
+        "sdBAL_deposit_contract": "0x3e0d44542972859de3CAdaF856B1a4FD351B4D2E",
+        "sdBAL_gauge_contract": "0x3E8C72655e48591d93e6dfdA16823dB0fF23d859",
+        "delegate_Registry": "0x469788fE6E9E9681C6ebF3bF78e7Fd26Fc015446"
```

this is actually snapshot's registry

```suggestion
        "sdBAL_gauge_contract": "0x3E8C72655e48591d93e6dfdA16823dB0fF23d859"
    },
    "snapshot": {
        "delegate_Registry": "0x469788fE6E9E9681C6ebF3bF78e7Fd26Fc015446"
```

---

ref: https://docs.snapshot.org/user-guides/delegation